package.json

```diff
@@ -124,7 +122,7 @@
     "rdf-parse": "^1.9.1",
     "rdf-serialize": "^1.2.0",
     "rdf-terms": "^1.7.1",
-    "redlock": "^4.2.0",
+    "redlock": "^5.0.0-beta.2",
```

Comment:
I know, I was also not a fan of this but here is the problem.
My RW locker uses ioredis and not redlock. The existing RedisResourceLocker uses redlock which uses ioredis. I need the newest version of ioredis, while the RedisReseourceLocker needs the 4.x version of ioredis. This is why I updated the RedisResourceLocker to use the newer redlock (so that I can use the newest ioredis without version conflicts)

Reply:
I understand the need, that's why I hope it comes out of beta soon so we can quickly get rid of it 😄

Reply:
Good news! https://github.com/luin/ioredis/releases/tag/v5.0.4

Reply:
Is removed because of removing RedisResourceLocker.